### PR TITLE
PWGGA: Add Helper Class for PHOS Trigger Mimicking

### PR DESCRIPTION
**Pull request by jluehder**
Adding a new Task, which checks events if they are triggered.
Also changed run ranges in AliPHOSTriggerUtils.cxx.
Changes to Trigger Mimick function, to use this class in PHOS

### CODE REVIEW

**Comment by kharlov** [comment]
+1

File: PWGGA/GammaConvBase/AliConvEventCuts.cxx [hunk 1]
```diff
@@ -5041,6 +5044,16 @@ Bool_t AliConvEventCuts::MimicTrigger(AliVEvent *event, Bool_t isMC ){
...
+  if (fSpecialTrigger == 6){
+    AliCaloTriggerMimickHelper* tempMimickHelper = 0x0;
+    tempMimickHelper = (AliCaloTriggerMimickHelper*) (AliAnalysisManager::GetAnalysisManager()->GetTask(Form("CaloTriggerHelper_%s", GetCutNumber().Data()) ));
+    if (tempMimickHelper){
+      return tempMimickHelper->GetEventChosenByTrigger();
+    } else {
+      return kFALSE;
+    }
+  }
+
```
Comment (nschmidtALICE) [inline PWGGA/GammaConvBase/AliConvEventCuts.cxx]: If this is the only way that trigger mimicking should work for PHOS, then please make it an AliFatal if the helper is not found.

File: PWGGA/GammaConvBase/AliConvEventCuts.h [hunk 2]
```diff
@@ -760,6 +765,7 @@ class AliConvEventCuts : public AliAnalysisCuts {
       TString                     fNameHistoReweightingMultMC;            ///< Histogram name for reweighting Eta
       TH1D*                       hReweightMultData;                      ///< histogram input for reweighting Eta
       TH1D*                       hReweightMultMC;                        ///< histogram input for reweighting Pi0
+      phosTriggerType             fPHOSTrigger;                           // Kind of PHOS trigger: L0,L1
```
Comment (nschmidtALICE) [inline PWGGA/GammaConvBase/AliConvEventCuts.h:768]: I guess the classdef needs increasing.

**Comment by jluehder** [comment]
I did change some minor things:
1) I changed to the correct author
2) I changed the name Mimick to Mimic, for correct english
3) I increades the Class Def in AliConvEventCuts

**Comment by jluehder** [comment]
I added AliFatals
For the new Mimic method the case fMimicTrigger == 3 is now needed

**Comment by nschmidtALICE** [comment]
+test

**Comment by nschmidtALICE** [comment]
+test

**Comment by nschmidtALICE** [comment]
+test

**Comment by nschmidtALICE** [comment]
+1

**Pull request merged**